pynopegl-utils/pynopegl_utils/viewer/__init__.py

```diff
@@ -89,6 +89,8 @@ def __init__(self, app, qml_engine, ngl_widget, args):
         self._ngl_widget.livectls_changed.connect(self._livectls_model.reset_data_model)
 
         self._script = app_window.findChild(QObject, "script")
+        if script.endswith(".py"):
+            script = QUrl.fromLocalFile(script).url()
```

Comment:
This is not going to cause a problem?
```python
>>> QUrl.fromLocalFile("file://foo/bar.py").url()
'file:file://foo/bar.py'
```

Reply:
The value stored in the config is not an URI and must remain a regular path.